src/App.tsx

```diff
@@ -1,9 +1,32 @@
-import React from 'react';
-import './App.scss';
+import React, { useEffect, useState } from 'react';
 import { MoviesList } from './components/MoviesList';
+import { Movie } from './types/Movie';
+
 import moviesFromServer from './api/movies.json';
+import './App.scss';
+
+const singleCaseQuery = (query: string) => {
+  return query.toLowerCase();
+};
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+  const [visibleMovies, setVisibleMovies] = useState<Movie[]>([]);
+
+  useEffect(() => {
+    setVisibleMovies(moviesFromServer);
+  }, []);
+
+  const filterList = (value: string) => {
+    setVisibleMovies(moviesFromServer.filter((movie) => {
```

Comment:
Don't set in state filtered movies in state. Create function to filter. Write results in variables and then use it. Also use `useMemo` for this vara